Incompatible with npm7

When installing this package with npm 7 it will error with the below message, this is because npm has [reduced the environment variables they apply](https://github.com/npm/rfcs/blob/latest/implemented/0021-reduce-lifecycle-script-environment.md) and the `npm_package_opencv` variable is now gone.

I haven't tested but it's possible this could be replaced with resolving `npm_package_from` and pulling the `opencv` property directly. If I can figure it out I'll send a PR your way

# Error Message
```txt
npm ERR! code 1
npm ERR! path /Users/awam/git/editors/vscode-appcelerator-titanium/node_modules/opencv4nodejs-prebuilt
npm ERR! command failed
npm ERR! command sh -c prebuild-install || (node ./install/dependencies.js && node ./install/install.js)
npm ERR! Installing prebuilt OpenCV vundefined for plattform darwin
npm ERR! npm ERR! code ETARGET
npm ERR! npm ERR! notarget No matching version found for @nut-tree/opencv-build-darwin@undefined.
npm ERR! npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! npm ERR! notarget a package version that doesn't exist.
npm ERR!
npm ERR! npm ERR! A complete log of this run can be found in:
npm ERR! npm ERR!     /Users/awam/.npm/_logs/2021-02-03T11_36_32_366Z-debug.log
npm ERR! child_process.js:674
npm ERR!     throw err;
npm ERR!     ^
npm ERR!
npm ERR! Error: Command failed: npm install @nut-tree/opencv-build-darwin@undefined
```
(using via vscode-extension-tester -> nut-js so unsure on most of the below)

*OpenCV version* (example 3.4.1): unsure

*With OpenCV-contrib?* (extra modules): unsure

*OS*: MacOSX

--- a/install/dependencies.js
+++ b/install/dependencies.js
@@ -2,7 +2,7 @@ const { execSync } = require("child_process");
 const isX64 = process.arch === "x64";
 
 const install = (pkg) => {
-    execSync(`npm install ${pkg}`);
+    execSync(`npm install --no-save ${pkg}`);
 }
 
 const packages = {
@@ -21,9 +21,18 @@ if (!process.env["OPENCV4NODEJS_PREBUILT_SKIP_DEPENDENCIES"]) {
     }
 
     const op = process.platform;
+    let openCvVersion =  process.env.npm_package_opencv;
 
-    console.log(`Installing prebuilt OpenCV v${process.env.npm_package_opencv} for plattform ${op}`);
-    install(`@nut-tree/opencv-build-${op}@${process.env.npm_package_opencv}`);
+    if (!openCvVersion) {
+        // npm 7 no longer sets fields from the package.json in process.env so lookup the package.json
+        // file we're running from. The environment variables are different when running locally vs
+        // as a dependency
+        const package = require(process.env.npm_package_from || process.env.npm_package_json);
+        openCvVersion = package.opencv;
+    }
+
+    console.log(`Installing prebuilt OpenCV v${openCvVersion} for platform ${op}`);
+    install(`@nut-tree/opencv-build-${op}@${openCvVersion}`);
     packages[op].forEach(pkg => {
         console.log(`Installing additional runtime dependency '${pkg}'`);
         install(pkg);

